common/JailUtil.cpp

```diff
@@ -156,8 +156,7 @@ void cleanupJails(const std::string& root)
         return;
     }
 
-    // FIXME: technically, the loTemplate directory may have any name.
-    if (FileUtil::Stat(root + "/lo").exists())
+    if (FileUtil::Stat(root + '/' + LO_JAIL_SUBPATH).exists())
```

Comment:
Not sure I like this one. What do we do about the cases where we can't chroot ? do these continue working ? do we have good tests for that ? That mode is really important for eg. OpenShift - and we should really have tests for it.

Reply:
I merely made "lo" a constexpr called LO_JAIL_SUBPATH (`constexpr const char LO_JAIL_SUBPATH[] = "lo";`). 

Really no idea about OpenShift and test coverage. There is no functional change here.